docs/devguide/contributing.asciidoc

```diff
@@ -47,16 +47,19 @@ Beats].
 The Beats are Go programs, so install the {go-version} version of
 http://golang.org/[Go] which is being used for Beats development.
 
-The location where you clone is important. Please clone under the source
-directory of your `GOPATH`. If you don't have `GOPATH` already set, you can
-simply set it to the `go` directory in your home (`export GOPATH=$HOME/go`).
+After https://golang.org/doc/install[installing Go], set the
+https://golang.org/doc/code.html#GOPATH[GOPATH] environment variable to point to
+your workspace location, and make sure the Go `bin` directory is in your PATH.
```

Comment:
Considering the level of detail we're shooting for, it might be best to say "make sure `$GOPATH/bin` is in your PATH..." or something similar as opposed to "the Go `bin` directory" which sounds a little vague.